CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to ci-imgs
+
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Report a bug or request a feature
+The best way to get in touch is by [opening an issue](https://github.com/rapidsai/ci-imgs/issues/new/choose).
+
+## To make a code contribution
+1. Fork the repo and create your branch from `main`.
+2. Make your changes.
+3. If you've changed APIs, update the documentation.
+4. Make sure your code lints.
+5. Issue that pull request!
+
+## Developer Certificate of Origin
+All contributions to this project must be accompanied by a sign-off indicating that the contribution is made pursuant to the Developer Certificate of Origin (DCO). This is a lightweight way for contributors to certify that they wrote or otherwise have the right to submit the code they are contributing to the project.
```

Comment:
Is this a new requirement? I am not familiar with this requirement from NVIDIA or other open-source projects.

To be clear, is this just "normal" git commit signing? (Like what we use to get the "Verified" tag on our commits?) If so, do we need to enforce something programmatically on our PRs?

Reply:
> In repos that have it, I've found it to be a frequent source of "ugh I forgot to do that, I need to force-push over what I just pushed" types of friction.

And those attestations are just text at the bottom of the commit message that anyone can write.  I don't think this offers anything more than proper commit signing does, and it introduces additional CI footguns.

Reply:
I should also note, DCO signoff is only needed by non-NV employees. Up to y'all if you want to include that note in the contributing doc.